types/chai/index.d.ts

```diff
@@ -545,6 +536,8 @@ declare namespace Chai {
          * @param message   Message to display on error.
          */
         isAbove(valueToCheck: number, valueToBeAbove: number, message?: string): void;
+        isAbove(valueToCheck: Date, valueToBeAbove: Date, message?: string): void;
+        isAbove(valueToCheck: unknown, valueToBeAbove: unknown, message?: string): void;
```

Comment:
What's the `unknown` overload for?  I'm sure the function does something, but does it make sense for anyone to call it that way?